src/main/java/com/adobe/testing/s3mock/domain/FileStore.java

```diff
@@ -741,5 +811,14 @@ private S3Object resolveS3Object(final String bucket, final String key) {
     }
     return s3Object;
   }
+
+  private static class MultipartUploadInfo {
```

Comment:
Please move `MultipartUploadInfo` to upper level.

Reply:
The reasoning for keeping it private to the `FileStore` was that it's an implementation detail of the `FileStore` and not a domain concept. Anyways, I can move it to a dedicated class file.